### PR TITLE
15 feature cities table

### PR DESCRIPTION
**Pull request by muginga72**

### CODE REVIEW

**Review by Ronapj1991 — APPROVED** [review]
Something is failing. Adding my approval but please check @muginga72

File: Gemfile [hunk 1]
```diff
@@ -27,7 +27,7 @@ gem "stimulus-rails"
 gem "jbuilder"
 
 # Use Redis adapter to run Action Cable in production
-# gem "redis", ">= 4.0.1"
+gem "redis", ">= 4.0.1"
```
Comment (Ronapj1991) [inline Gemfile:30]: Curious.. What is redis for?

Comment (muginga72) [reply Gemfile:30]: Redis its a gem that supports geocode gem but we don't need it anymore. I'll remove it.

**Comment by muginga72** [comment]
I am closing the PR due to failing on tests.

**Pull request closed without merging**